utils/data_utils.py

```diff
@@ -34,13 +34,27 @@ def get_engine(stage: Literal["dev", "prod"] = "dev"):
 
 def fetch_flood_data(pcode, adm_level):
     """Fetch flood exposure and administrative data from database."""
-    query_exposure = text(
-        """
-        SELECT *
-        FROM app.floodscan_exposure
-        WHERE pcode=:pcode AND adm_level=:adm_level
-        """
-    )
+    if adm_level == "region":
+        parts = pcode.split("_region_")
```

Comment:
A bit clunky yes, but does the job! We'll improve this as we start to work on custom regions more down the line